src/libs/SearchUIUtils.ts

```diff
@@ -1300,46 +1483,22 @@ function isCorrectSearchUserName(displayName?: string) {
 }
 
 function createTypeMenuSections(
-    currentUserLogin: string | undefined,
+    currentUserEmail: string | undefined,
     currentUserAccountID: number | undefined,
     feeds: OnyxCollection<OnyxTypes.CardFeeds>,
+    defaultCardFeed: string | undefined,
```

Comment:
Passing both `feeds` and `defaultCardFeed` creates some duplicated work as now we are calling `getCardFeedsForDisplayPerPolicy` twice. Can we do the following:

1. Rename `useDefaultCardFeed` to `useCardFeedsForDisplay`
2. `useCardFeedsForDisplay` should return both `defaultCardFeed` as it does now but it should be of type `CardFeedForDisplay` (then you can use defaultCardFeed.id in suggested searches), and it should also return `cardFeedsPerPolicy`
3. `createTypeMenuSections` should accept `defaultCardFeed: CardFeedForDisplay | undefined` and `cardFeedsPerPolicy: Record<string, CardFeedForDisplay[]>`